bezier-rs/docs/interactive-docs/src/utils/drawing.ts

```diff
@@ -41,23 +45,22 @@ export const drawText = (ctx: CanvasRenderingContext2D, text: string, x: number,
 	ctx.fillText(text, x, y);
 };
 
-export const drawBezierHelper = (ctx: CanvasRenderingContext2D, bezier: WasmBezierInstance, stroke = "black", radius = 5): void => {
+export const drawBezierHelper = (ctx: CanvasRenderingContext2D, bezier: WasmBezierInstance, stroke = "black", radius = DEFAULT_ENDPOINT_RADIUS): void => {
 	drawBezier(
 		ctx,
-		bezier.get_points().map((p) => JSON.parse(p)),
+		bezier.get_points().map((p: string) => JSON.parse(p)),
```

Comment:
Can you set this long line to a variable on the line above in order to make this function call not require so many lines?